setup.sh

```diff
@@ -47,33 +47,24 @@ function check_dependencies()
   local pip_package_list=''
   local cmd=''
   local distro
-
+  debian=("apt list --installed | cut -d '/' -f1" "apt install -y")
```

Comment:
Use `--delimiter=DELIM` instead of `-d`. Additionally, if you don't have a string concatenation, I recommend using single-quotes (review this in this entire commit).